docs/releases/5.0.4.txt

```diff
@@ -13,3 +13,10 @@ Bugfixes
   fields with expressions in ``db_default``. As a consequence,
   ``Model.full_clean()`` no longer validates for empty values in fields with
   ``db_default`` (:ticket:`35223`).
+
+* Fixed a regression in Django 5.0 where the ``AdminFileWidget`` template would
+  render the ``id`` attribute twice on the clear checkbox (:ticket:`35273`).
+
+* Fixed a regression in Django 5.0 where ``ClearableFileInput``'s
+  ``checked`` attribute is not preserved on invalid forms (:ticket:`35273`),
+  (:ticket:`34488`).
```

Comment:
Is this a regression? It looks like a case that we missed when fixing ticket-34488 not a regression, so it should be handled separately and applied only to the `main` branch. I can reorganize commits accordingly if you want, and remove this release note.

Reply:
Hm… yes, if non-regression bugs not only applied to a future release, yes. I'd appreciate the help, since you know best what to apply where.